docs/sources/reference/components/prometheus/prometheus.exporter.windows.md

```diff
@@ -268,6 +268,15 @@ debug information.
 `prometheus.exporter.windows` does not expose any component-specific
 debug metrics.
 
+[wrap-regex]: #wrapping-of-regex-strings
+## Wrapping of regex strings
```

Comment:
I recommend to add some notes and example and consequences here.

Regex as case sensitive. For case in sensitive match, use `(?i)` as prefix. 

Secondly, an example for the regex wrapping

Normally, a regex like `e.*` would match `email` and `design`. To avoid that, all regex are wrapped (as explained). If the original behavior is required, `.*` must be used as prefix, `.*e.*` would  `email` and `design` again.

Reply:
> BTW according to [regex101.com](https://regex101.com/), e.* matches esign whereas ^(?:.*e.*)$ matches design - so they are not exactly equivalent?

But alloy wraps `e.*` into `^(?:e.*)$` which would not match design anymore. 

Ref: https://go.dev/play/p/yf5gxEZ3oeZ

Reply:
I could add this note?
```
{{< admonition type="note" >}}
The wrapping may change the behaviour of your regex.
For example, the `e.*` regex would normally match both the "service" and "email" strings.
However, `^(?:e.*)$` would only match "email".
{{< /admonition >}}
```